phpBB/includes/functions_download.php

```diff
@@ -223,9 +223,6 @@ function send_file_to_browser($attachment, $category)
 		}
 	}
 
-	// Close the db connection before sending the file etc.
-	file_gc(false);
```

Comment:
why moving it?

Reply:
Read_stream use database to check if the file exist